cvat/apps/dataset_manager/formats/transformations.py

```diff
@@ -32,3 +35,16 @@ def transform_item(self, item):
                 z_order=ann.z_order))
 
         return item.wrap(annotations=annotations)
+
+class EllipsesToMasks(ItemTransform):
```

Comment:
```suggestion
class EllipsesToMasks:
```
There is no point in making it inherited now.